### PR TITLE
Update to python3 deps

### PR DESCRIPTION
**Pull request by tomflynn**
Point to bionic instead of xenial

Signed-off-by: Tom Flynn <tom.flynn@gmail.com>

### CODE REVIEW

**Comment by snaiksat** [comment]
have we tried building a deb with these changes?

File: provision/debian/control [hunk 1]
```diff
@@ -4,14 +4,14 @@ Priority: optional
 Maintainer: Cisco Systems, Inc. <dhami@noironetworks.com>
 X-Python-Version: >= 3.0
 Build-Depends: debhelper (>= 9), python-setuptools (>= 3.3), python3-all, dh-python
-Standards-Version: 3.9.5
+Standards-Version: 4.1.4
```
Comment (apoorva11029) [inline provision/debian/control:7]: Just to confirm, have you reviewed any changes that need to be done for this standards' upgrade?
Based on: https://www.debian.org/doc/debian-policy/ch-controlfields.html#s-f-standards-version and https://www.debian.org/doc/debian-policy/ch-source.html#s-standardsversion

**Comment by tomflynn** [comment]
> have we tried building a deb with these changes?

Yes

File: provision/debian/control (hunk 1 above)
Comment (tomflynn) [reply provision/debian/control:7]: No. lintian was complaining so I bumped the version and there were no new lintian complaints. The resulting deb installs fine on Ubuntu 18.04

**Review by apoorva11029 — APPROVED** [review]

**Pull request merged**